### DISCUSSION TITLE
Typst definition lists with multiple definitions – include a line break / separator

### DISCUSSION THREAD

**Discussion by baptiste** [jgm/pandoc#9704] [General]
I've noticed that the output from pandoc with the compact definition list example

```
Term 1
  ~ Definition 1

Term 2
  ~ Definition 2a
  ~ Definition 2b
```

results in 

```
/ Term 1: #block[
Definition 1
]

/ Term 2: #block[
Definition 2a
Definition 2b
]

```

where unfortunately the line break between the two definitions is not usable, as typst would require an explicit `\` to break the lines. Running the typst output through pandoc with md output shows that it's not semantically present:

```
/ Term 1:

Definition 1

/ Term 2:

Definition 2a Definition 2b
```

I'm hoping to use the typst output as separate lines, but currently I don't think it's possible.

**Comment by jgm** [comment]
Actually this would work:
```
/ Term 2: #block[
Definition 2a

Definition 2b
]
```
I will make that change.

**Comment by jgm** [comment]
I've fixed this in commit 5b9a8ea99cf93b1529672080324ac95c8b0254d0

**Reply by baptiste** [reply]
Brilliant, thank you!
